frame/composable-traits/src/clearing_house.rs

```diff
@@ -38,13 +45,40 @@ pub trait ClearingHouse {
 	/// ## Returns
 	/// The new market's id, if successful
 	fn create_market(config: &Self::MarketConfig) -> Result<Self::MarketId, DispatchError>;
+
+	/// Open a position in a market
+	///
+	/// This may result in the following outcomes:
+	/// * Creation of a whole new position in the market, if one didn't already exist
+	/// * Increase in the size of an existing position, if the trade's direction matches the existion position's one
```

Comment:
"Increase _in_ the size...", does this particle need to be present or would be more natural to just omit it?
(I'm not sure about it, but seems a bit off.)

Reply:
Not sure TBH. The way I thought about it was "A possible outcome is an increase _in/of_ the size of an existing position"